feature/course-view/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/courseview/ui/LectureListUi.kt

```diff
@@ -45,7 +50,9 @@ internal fun LectureListUi(
             getItemId = { id ?: 0L }
         ) { lecture ->
             LectureListItem(
-                modifier = Modifier.fillMaxWidth(),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(8.dp),
```

Comment:
This applies an 8dp horizontal padding and a 16dp vertical padding between list items. Is this really what is required here?

Reply:
Has been resolved by using the vertical arrangement in WeeklyItemsLazyColumn like it is used in ExerciseListUi.